src/browser/lib/Levels/level7.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Level 7
+ *
+ * 26 Enemies
+ * 14 Ammo
+ * 07 Health
+ * 07 Stamina
+ */
+export const Level7 = [
+  'W W W W W W W W W W W W W W W W W W W W W W W W W W',
+  'W               W                 W   W   W   W   W',
+  'W   A A   P     W   E   W   E     W W W   H   A   W',
+  'W               W   A   W   A   A   W   A   W   A W',
+  'W W W W W W W W W   S   W   S   S W W W   A   S   W',
```

Comment:
Maybe create an opening here e.g.

![image](https://github.com/sentrychris/undeadbytes/assets/42321445/2268cff0-c331-4016-ab25-2370ec5676c5)

Reply:
Ok sounds good I can fix those changes and do a new pull request.